src/usr/local/www/services_unbound.php

```diff
@@ -435,6 +437,14 @@ function build_if_list($selectedifs) {
 	    'in "tun" mode. The domain in %sSystem: General Setup%s should also be set to the proper value.',
 	    '<a href="system.php">','</a>'));
 
+$section->addInput(new Form_Checkbox(
+	'replace1to1',
+	'1:1 NAT Addresses',
+	'Replace 1:1 NAT IP Addresses in Responses',
+	$pconfig['replace1to1']
+))->setHelp('If this option is set, then when an A record is returned with an IP address from the 1:1 NAT table, the value will be replaced'.
+					' with the internal IP address. This is an alternative to manually creating split-DNS.');
```

Comment:
It seems like it's worth having a strong note here stating that this activates the `respip` module which inspects and alters DNS responses directly, which can degrade performance.

I could see some users having privacy or performance concerns with that, but so long as there is a warning there it should be fine.

Reply:
Help text improved.